### DISCUSSION TITLE
How can I Map Fields Which are Not in Entity, like Audit fields while Inserting and Updating

### DISCUSSION THREAD

**Discussion by kuttikrishnankodoth** [nhibernate/fluent-nhibernate#434] [General]
Hello Team,
I am developing an application on top of a legacy Oracle Db, all the tables has got audit fields like Created Dt, Last UpdtedDt, Created User and Last Updated and all of them are mandatory fields as well by data base set up. .But I don't want to bring these fields in to my entity  since it don't have any domain value, is there any way that i can set values to this fields without adding them in to my entities ?
i found some articles about preinsert events, but what i under stood is that also require me to add the property in the domain.
Any help will be appreciated.
Thanks in Advance.

**Comment by jlscpns** [comment]
Perhaps Stack Overflow would be a better suited place for this kind of question and while I'm not aware of a solution getting around the need to map columns, in case it hasn't crossed your mind, collecting all the columns to a [Component(Map)](https://github.com/FluentNHibernate/fluent-nhibernate/wiki/Fluent-mapping/27b08a8714f8e81917f912d208f719e5fad1fbed#components) could lessen the pain, especially if all the tables conform to the same structure. Some interface magic could be done as well to help out further if it seems worth it.
